setup.py

```diff
@@ -1,47 +1,24 @@
-from re import search
 from setuptools import setup, find_packages
 
-
-PATH = 'candies/cli'
-
-
-def text_of(path):
-    with open(path, 'rt', encoding='utf8') as file:
-        return file.read()
-
-
-def version():
-    text = text_of(f'{PATH}/__init__.py')
-    match = search(r"__version__ = \'(.*?)\'", text)
-
-    return match.group(1)
-
-
-def readme():
-    return text_of('README.md')
-
-
-def requirements():
-    return text_of('requirements.txt')
-
-
 setup(
     name='candy-cli',
-    version=version(),
+    version='0.0.1',
+    packages=find_packages(exclude=['tests*']),
+    license='MIT',
     description='Tools for writing clean CLI code.',
-    long_description=readme(),
+    long_description=open('README.md').read(),
```

Comment:
Shouldn't we close the file after reading?

Reply:
I can think of a case when file will not close automatically, for example, when using a non-CPython interpreter, but for CPython there is no reference to a file after `open('README.md')`, so it will be closed right after `read()`.

Personally, I like the way `setup.py` looks without any additional functions. Do you think we need to read more accurately here?

Reply:
Oh, if you are sure that the file will be closed almost immediately, then I am okay with this. I believe that it is my experience with C# and finalizers that makes it feel like something is wrong with code that is not using `with` statements. :)

Reply:
Great, thanks for the paying attention!